### PR TITLE
Uploading latest Tau SF functions

### PR DESCRIPTION
**Pull request by anpicci**
This PR updates the tau SF implementation to the latest version available

### CODE REVIEW

**Comment by bryates** [comment]
@anpicci please fix the [lint errors](https://github.com/TopEFT/topeft/actions/runs/8695007851/job/23845114599?pr=408#step:4:32)

File: topeft/modules/corrections.py
Comment (bryates) [inline topeft/modules/corrections.py:404]:
```suggestion

```

**Review by bryates — CHANGES_REQUESTED** [review]
Thanks @anpicci. Overall this is a simple change, but since this relies on changes to TopCoffea (https://github.com/TopEFT/topcoffea/pull/26), we should either update the CI to temporarily check out the `TauSF_update`, or wait until that is merged and then rerun the CI here bem

**Comment by anpicci** [comment]
Ok, what solution do you prefer @bryates ?

**Comment by bryates** [comment]
> Ok, which solution do you prefer @bryates ?

I'm leaning towards updating the CI to checkout the temporary branch, so we can get it all running and maybe merge now. That does of course mean we'll have to fix it again later. @Andrew42 and @sscruz what do you think?

**Comment by anpicci** [comment]
I was wondering if there is any update on this @bryates

**Comment by bryates** [comment]
> I was wondering if there is any update on this @bryates

@anpicci while we're waiting for the TopCoffea PR, can you fix these lint errors?
https://github.com/TopEFT/topeft/actions/runs/8706515199/job/23879255324?pr=408#step:4:32
It looks like just 2 lines with blank spaces at the end.

**Review by bryates — APPROVED** [review]
Now that the TopCoffea PR has been merged, this PR should be ready. I've merged the laste changes from `master`, and the CI is running now. @anpicci did you have any other updates for this PR?

**Comment by anpicci** [comment]
@bryates no, I don't need to add anything else, thanks!

**Comment by anpicci** [comment]
hi @bryates , do you think we can merge?

**Comment by bryates** [comment]
> hi @bryates , do you think we can merge?

I wanted to make sure @andrea21z and @sscruz don't have any comments, but I'm happy with this PR.

**Comment by anpicci** [comment]
I'm afraid @andrea21z is not working for CMS anymore

**Comment by bryates** [comment]
> I'm afraid @andrea21z is not working for CMS anymore

I meant @Andrew42 😂

**Pull request merged**
